parts/language/lang_vi.lua

```diff
@@ -883,7 +883,7 @@ return {
         ['blind_h']=        {"Invisible",         "KHÓ",         "Dành cho người đã có kinh nghiệm"},
         ['blind_l']=        {"Invisible",         "KHÓ+",        "Dành cho người chơi chuyên nghiệp"},
         ['blind_u']=        {"Invisible",         "?",              "Bạn đã sẵn sàng chưa?"},
-        ['blind_wtf']=      {"Invisible",         "WTF",            "Bạn chưa chuẩn bị cho màn này đâu!"},
+        ['blind_wtf']=      {"Invisible",         "CLGT?",            "Bạn chưa chuẩn bị cho màn này đâu!"},
```

Comment:
I think we can use "**C**ái **Q**uái **G**ì **Đ**ây **?**" may be better<br>
But you can keep your change

Reply:
I think CLGT? is better since it has the same vibe as WTF?

Reply:
"CLGT" has the same literal style as "WTF" while also containing a swear word in there as well

Reply:
I could also came with "Cái Đ** Gì Đây?" since it was a common swear word used by Vietnamese teenagers